### ISSUE TITLE
The `node.position` data is lost after using the `rehype-prism-plus@1.4.2` plugin.

### ISSUE THREAD

**Issue by jaywcjlove** [timlrx/rehype-prism-plus#46]
- https://github.com/uiwjs/react-markdown-preview/issues/195#issue-1297472301

The `node.position` data is lost after using the `rehype-prism-plus@1.4.2` plugin. @timlrx 

```jsx
<MarkdownPreview
  components={{
    code:  ({ inline, node, ...props }) => {
      if (inline) {
        return <code {...props} />;
      }
      console.log('>>>>does not exist!!!!:::', node.position)
      // `node.position` does not exist
      // position:
      //   end: {line: 44, column: 35, offset: 2393}
      //   start: {line: 44, column: 30, offset: 2388}
      return (
        <div></div>
      )
    }
  }}
  className="App-markdown"
  source={MDStr.replace(/([\s\S]*)<!--dividing-->/, '')}
/>
```

**Comment by timlrx** [comment]
In the meantime, do you mind fixing your dependency to 1.4.1 first? Thanks.

**Comment by jaywcjlove** [comment]
`1.4.1` is working. @timlrx 
The `position` object is important to me.

**Comment by timlrx** [comment]
I pushed a new fix which adds the `column` field required by the node position, which seems to work fine with @mdx/loader. This means I can remove `recursivelyStripPositions` and restore node positions. This is published as v1.4.3. Let me know if you run into issues.

**Comment by jaywcjlove** [comment]
Thanks! I will take the time to test it. @timlrx
